.github/workflows/pull-request-checks.yaml

```diff
@@ -20,6 +20,11 @@ jobs:
           sudo apt-get install --no-install-recommends -yq gcc gdb g++ maven jq flex bison libxml2-utils ccache cmake z3
       - name: Confirm z3 solver is available and log the version installed
         run: z3 --version
+      - name: Download cvc-5 from the releases page and make sure it can be deployed
+        run: |
+          wget https://github.com/cvc5/cvc5/releases/download/cvc5-0.0.7/cvc5-Linux
```

Comment:
Would it be possible to have a macro for the version number because I know the cvc5 release 1.00 is due soon?

Reply:
Hi Martin, I'll have a look to see if there's something I can do for this.

I think a macro may not be possible (in yaml), but maybe it's possible to store the version in a variable and do some string interpolation, which is going to concentrate the knowledge for the version in just one place, allowing for an easy update.

Reply:
Thanks! Any of these would be great, parallel changes to multiple instances of the same string always makes me nervous.